tests/php/core-api/wpcom-fields/test-post-fields-publicize-connections.php

```diff
@@ -38,6 +38,29 @@ public static function wpSetUpBeforeClass( $factory ) {
 
 		self::$user_id = $factory->user->create( array( 'role' => 'administrator' ) );
 
+		// Hacky :(
```

Comment:
I wouldn't leave this comment here. Sometimes we do hacky things and there is no shame about that. :)

Reply:
Yeah, I agree. This is not the single place we're doing this, and it only makes sense when we have different behavior in different envs 😉